deps/bypy/bypy.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "bypy",
+	"buildsystem": "simple",
+	"build-commands": [
+		"mkdir -p \"${FLATPAK_DEST}/lib/python3.8/site-packages\"",
+		"cp -avr bypy \"${FLATPAK_DEST}/lib/python3.8/site-packages\"",
+		"install -T __main__.py /app/bin/bypy"
+	],
+	"cleanup": ["*"],
+	"sources": [
+		{
+			"sha512": "9271c26e64cc47de446c206ddf34615c123be5c2cbff69e18a1f84632a2e498f72750f9d1078a40556c9f626a29befd5d0f849df9323b803000338f14b12d104",
+			"type": "archive",
+			"url": "https://github.com/kovidgoyal/bypy/archive/e0811601269216758ecb9988b4f21d6b56419bbc.zip"
```

Comment:
Instead of downloading the archive, can this be replaced by a 'Directory Source' (dir type) since we've already got it as a submodule?

Reply:
I decided not to do this as it will completely break caching of the `flatpak-builder` build process since `bypy` is the very first module being built. Additionally, bypy has quite a few patches that are not in the submodule.

Reply:
If it is OK, I'd prefer to edit the Flathub file to not download the submodule when performing builds as it is not needed there?